contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/expectations/expect_column_values_are_probabilistically_greater_than_or_equal_to_threshold.py

```diff
@@ -0,0 +1,176 @@
+                    "McRoomyRoom",
+                ],
+                "INSTURL": [
+                    "www.aamu.edu/",
+                    "https://www.uab.edu",
+                    "www.amridgeuniversity.edu",
+                    "McRoomyRoom",
+                ],
+                "NPCURL": [
+                    "www.aamu.edu/admissions-aid/tuition-fees/net-price-calculator.html",
+                    "https://uab.studentaidcalculator.com/survey.aspx",
+                    "www2.amridgeuniversity.edu:9091/",
+                    "McRoomyRoom",
+                ],
+                "LATITUDE": ["34.783368", "33.505697", "32.362609", "McRoomyRoom"],
+                "LONGITUDE": ["-86.568502", "-86.799345", "-86.17401", "McRoomyRoom"],
+                "RELAFFIL": ["NULL", "NULL", "74", "McRoomyRoom"],
+                "DEATH_YR2_RT": [
+                    "PrivacySuppressed",
+                    "PrivacySuppressed",
+                    "PrivacySuppressed",
+                    "McRoomyRoom",
+                ],
+                "SEARCH_STRING": [
+                    "Alabama A & M University AAMU",
+                    "University of Alabama at Birmingham ",
+                    "Amridge University Southern Christian University  Regions University",
+                    "McRoomyRoom",
+                ],
+            },
+            "tests": [
+                {
+                    "title": "positive_test_with_column_one",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {"column": "OPEID6", "threshold": 0.00},
+                    "out": {
+                        "success": True,
+                        "unexpected_index_list": [],
+                        "unexpected_list": [],
+                    },
+                },
+                {
+                    "title": "error_test_threshold",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {"column": "ZIP", "threshold": 0.61},
+                    "out": {
+                        "success": False,
+                    },
+                },
+            ],
+        }
+    ]
+
+    # This is the id string of the Metric used by this Expectation.
+    # For most Expectations, it will be the same as the `condition_metric_name` defined in your Metric class above.
+    map_metric = (
+        "column_values.prediction_confidence_greater_than_or_equal_to_threshold"
+    )
+
+    # This is a list of parameter names that can affect whether the Expectation evaluates to True or False
+    success_keys = (
+        "threshold",
+        "mostly",
+    )
+
+    # This dictionary contains default values for any parameters that should have default values
+    default_kwarg_values = {
+        "threshold": None,
+        "mostly": None,
+        "result_format": "BASIC",
+        "include_config": True,
+        "catch_exceptions": False,
+    }
```

Comment:
Hey @taylorfturner ! This is why you're seeing those failing checks in the diagnostics checklist.

When you provide mostly, you're explicitly saying "see if at least x % of my data passes this benchmark." Specifying None as the default value and not passing in a value for mostly in your tests means this comparison is attempted and fails unless the result of your expectation is also None.

Resolution here is one of:

removing mostly from your default_kwarg_values
specifying a different default value for mostly (typically 1) in your default_kwarg_values

Leaving it as None and specifying mostly with a valid value as a key under in in each of your tests would resolve your failing tests, but could lead to some ambiguous failing down the road if a user doesn't pass a valid value for mostly.

This is the same for the other expectation as well.

Reply:
Excellent -- thanks for the explanation. commits to resolve incoming!